app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -221,6 +222,46 @@ sealed class Event {
             get() = hashMapOf(TopSites.swipeCarouselKeys.page to page.toString())
     }
 
+    data class SecurePrefsExperiment(
```

Comment:
You'll need to change this classes to match the new events (see comments in metrics.yaml).

Make sure to set the extras explicitly, they'll be generated by glean from the metrics.yaml definitions (see code sample in https://mozilla.github.io/glean/book/user/metrics/event.html)